straxen/plugins/position_reconstruction.py

```diff
@@ -108,12 +110,8 @@ def _get_model_file_name(self):
 
 @export
 @strax.takes_config(
-    strax.Option("file_mlp",
-                 help="Name of saved MLP model file in the aux file data base."
-                      "The file contains both structure and weights. Set to "
-                      "None to skip the computation of this plugin.",
-                 default="xnt_mlp_wfsim_20201214.tar.gz"
-                 )
+    strax.Option('mlp_model',

```

Comment:
can you:

1. reinstate the None explanation? 
2. also for GCN and CNN.
3. Furthermore, we should add True to the tuple.
4. Set the default to ('mlp_model', 'ONLINE', True), we'd want to be using this from the start.

Reply:
done